Documentation/README.md

```diff
@@ -11,7 +11,22 @@
 
 ### System Instructions:
 0. Login to the Application - (https://team4.uwaterloo.ca/login/)
-1. [Review Applicants]
+1. [Change Application Form]
+	* Function: This page allows the AC to change the text on two pages, the Application Intro Page (https://team4.uwaterloo.ca/taform/) and the Application Page (https://team4.uwaterloo.ca/taform/application.html).
+	* Scenarios for Use:
+		* 1 - Application guidelines have changed.
+		* 2 - The term needs to be changed.
+		* 3 - A reference link has become outdated.
+	* How to Use:
+		* Login to the Application - (https://team4.uwaterloo.ca/login/)
+		* Open Step 4, Change Application Form - (https://team4.uwaterloo.ca/taform/upload_front_matter.html)
+		* For each file you may change (Intro Page, Front Matter), review the current page.
+		* Download the file you would like to change.
+		* Make changes within the downloaded file. Ensure you are using HTML syntax and save the file as .txt.
+		* Select the file you would like to upload.
+		* Upload the file.
+		* Review the changes by previewing the page you changed again.  
```

Comment:
I would add:

to test their page locally.

They can re save their txt file with the html extension and open it in the browser. (from ada's process)
But must upload with txt extension.